addon/components/table.ts

```diff
@@ -164,76 +161,36 @@ class TableComponent extends Component<TableArgs> {
         return !this.isLoading;
     }
 
-    /**
-     * CP that returns whether the table rows are clickable
-     *
-     * @type {boolean} clickableRows
-     * @memberof TableComponent
-     */
```

Comment:
i might keep this doc block comments on these getters (just tweaking the description to get rid of "CP") just to have the extra documentation as to what they're for